server/controllers/finance/reports/debtors/annual_clients_report.js

```diff
@@ -291,8 +295,11 @@ function getTotalsFooter(fiscalYearId, currencyId, rate, filterDebtorInsolvant,
     }
 
     if (filterDebtorConventioned) {
-      const filterValue = filterDebtorConventionedValue !== undefined ? filterDebtorConventionedValue : 0;
-      filterConventioned = ` AND dg.is_convention = ${filterValue}`;
+      if (filterDebtorConventionedValue !== undefined) {
+        filterConventioned = ` AND dg.is_convention = ${filterDebtorConventionedValue}`;
```

Comment:
Please confirm that this isn't passed from the client.  If it is passed from the client, it should be escaped using `db.exec()` or something similar.

Reply:
Same question.